docs/design/security/02-jet-job-security.md

```diff
@@ -0,0 +1,176 @@
+---
+title: 021 - Job Security
+description: Authorize jobs in a secure environment
+---
+
+## Summary
+
+Hazelcast provides fine-grained authorization mechanisms for distributed
+data structures. After merge with Hazelcast IMDG, we want to provide
+this mechanism for Jet jobs too.
+
+## Security in the Open Source Part
+
+- Disable Jet: One can disable Jet engine completely using
+`JetConfig#enable`. We don't create the internal Jet service and don't
+start the threads. If a user tries to obtain `JetService`, we throw an
+exception. Jet is enabled by default.
```

Comment:
AFAIU, "Jet is enabled by default" will be stale

Reply:
From my understanding, it's disabled by default only for the JAR file (embedded mode).